### PR TITLE
Mail Service-First Version

### PR DESCRIPTION
**Pull request by gaocegege**
CR need @gaocegege @at15

### CODE REVIEW

File: .ayi.yml [hunk 1]
```diff
@@ -1,2 +1,9 @@
----
-debug: true
\ No newline at end of file
+debug: true
+mailgun:
+  domain: mg.gaocegege.com
+  prikey: key-c59d88eb312de2549ce903119cf9ca21
```
Comment (at15) [inline .ayi.yml]: is it ok to put the key here? 0.0

Comment (gaocegege) [inline .ayi.yml]: = =逗比了，不过没关系，是free的account

**Comment by at15** [comment]
lgtm, \w/

File: .ayi.yml (hunk 1 above)
Comment (at15) [inline .ayi.yml]: ok.... travis 我记得可以设置 key 之类的

**Comment by gaocegege** [comment]
hang, try to solve secret problem

**Pull request closed without merging**